test/backend/mcode/TreatmentOptionsOutcome.test.js

```diff
@@ -4,55 +4,123 @@ import Enzyme, { shallow, mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-15';
 
 import TreatmentOptionsOutcome from '../../../src/mcode-pilot/components/TreatmentOptionsOutcomes/TreatmentOptionsOutcomes.jsx';
-import testPatients from './mock-data/testpatients.json';
+import {
+    includedTreatmentData,
+    comparedTreatmentData,
+    similarPatientTreatments
+} from './mock-data';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 describe("TreatmentOptionsOutcome", () => {
     let props, optionsOutcome;
 
-    const outcome = () => {
-      if (!optionsOutcome) optionsOutcome = mount(<TreatmentOptionsOutcome {...props} />);
-      return optionsOutcome;
-    };
-
     beforeEach(() => {
-      props = { similarPatients: testPatients.testGroup1 };
-      optionsOutcome = null;
+        props = {
+            comparedTreatments: ['chemotherapy', 'hormonal'],
+            includedTreatments: ['surgery', 'radiation'],
```

Comment:
This is probably going to cause problems with the tests when it switches to combination instead of comparison, there will be nothing in the view.

Reply:
When we make that change, we'll update the tests.